build.zig

```diff
@@ -68,8 +68,9 @@ pub fn build(b: *Builder) !void {
     exec.setTarget(target);
 
     const make_iso = switch (target.getCpuArch()) {
-        .i386 => b.addSystemCommand(&[_][]const u8{ "./makeiso.sh", boot_path, modules_path, iso_dir_path, exec.getOutputPath(), output_iso }),
-        .aarch64 => b.addSystemCommand(&[_][]const u8{ "aarch64-linux-gnu-objcopy", exec.getOutputPath(), "-O", "binary", try fs.path.join(b.allocator, &[_][]const u8{ exec.output_dir.?, "kernel8.img" }) }),
+        // .i386 => b.addSystemCommand(&[_][]const u8{ "./makeiso.sh", boot_path, modules_path, iso_dir_path, exec.getOutputPath(), output_iso }),
+        // .aarch64 => b.addSystemCommand(&[_][]const u8{ "aarch64-linux-gnu-objcopy", exec.getOutputPath(), "-O", "binary", try fs.path.join(b.allocator, &[_][]const u8{ exec.output_dir.?, "kernel8.img" }) }),
+        .aarch64 => b.addInstallRaw(exec, "kernel8.img"),
```

Comment:
Nice, I didn't know this existed.

Reply:
I will update build.zig to only improve objcopy vs. addInstallRaw. I think there is a type conflict in the switch statement between step types that I will resolve.

Reply:
Sounds good. This is a nice change.

Reply:
Actually I cannot easily figure out how to resolve the type conflict so I will make a separate pr for b.addInstallRaw().